src/Features/LanguageServer/Protocol/Handler/Definitions/AbstractGoToDefinitionHandler.cs

```diff
@@ -76,7 +77,6 @@ protected async Task<LSP.Location[]> GetDefinitionAsync(LSP.TextDocumentPosition
 
             return locations.ToArrayAndFree();
 
-            // local functions
```

Comment:
i won't block on this.  but I do like these comments as it makes things easier to read for me :)